docs/data/pages.ts

```diff
@@ -140,9 +140,10 @@ const pages: MuiPage[] = [
           {
             pathname: '/x/react-data-grid/server-side-data-group',
             title: 'Server-side data',
+            unstable: true,
```

Comment:
It's not stable.

Reply:
We don't use this convention for many other features like [`unstable_rowSpanning`](https://master--material-ui-x.netlify.app/x/react-data-grid/row-spanning/) for instance.
Or by "not stable" do you mean some of the features are missing?

Reply:
I like to have `newFeature: true` instead. Wdyt?

Reply:
> Or by "not stable" do you mean some of the features are still being developed?

@MBilalShafi I mean that the code has `unstable_` so the API can change without following semver, so it's API unstable.

> I'd like to have newFeature: true instead.

What the rational?

> We don't use this convention for many other features like unstable_rowSpanning for instance.

Oh I didn't noticed this one https://mui.com/x/react-data-grid/row-spanning/. Is this an exception?

https://mui.com/x/react-data-grid/list-view/ is using the unstable flag, that felt to match the product objective. It needs to be clear its unstable. API might change, code might be buggy. It's not GA (Generally Available) because we can't yet guarantee people will have a great experience. So we don't want too many people to use this. Only enough people who are desperate for it to try and give us feedback. 

When I see "New", my intuition points me toward assuming that it's ready to use, so that it's stable. `unstable: true,` makes the docs match the code, this seems right.

Reply:
I discussed this with the team. Will summarize the discussion below:

There seem to be different opinions on what has been perceived against an `unstable_` prefix for a flag or method, and experimental flag.

**POV 1:** The code is stable, but since it has just been released and not tested with a wider audience, let's release it with an `unstable_` prefix to have a margin of changing the API if some soft corners are found. (We did this with column grouping and it seemed to remain in an `experimental` state for 1.5y+)

It seems to defeat its purpose though, if `unstable_` is perceived as something not "safe" to be widely used, we would not get proper data points with wider audience testing.

**POV 2:** Similar to your point, the general perception of having an `unstable_` is closer to being "Preview" than being "New", which strengthens your point of having a criterion like "Generally Available" to be able to decide when a feature is ready to be widely used without much hiccups expected.

It connects to another point, should we mark every new feature as `unstable_` (we used to do this currently to have a space for breaking the API if needed)?

I would say "no". It should depend on the particular feature and its level of stability and further development plans to decide whether to mark it as `unstable` at all. Row spanning could be an example of that, it seems more standalone to be released as "New" from the beginning.

It'd be great to have a defined pattern that all the products would target to use. Some input from the dev-ex team would be great in this regard.

@joserodolfofreitas Feel free to add something

Reply:
> POV1 and POV2

I'm not perceiving the difference between those two, they feel like it's about the same thing: to be iterative, we ship, fail, iterate, and release as stable.

> if unstable_ is perceived as something not "safe" to be widely used, we would not get proper data points with wider audience testing.

I don't recall one case where an important problem wasn't reported during the unstable API phase but reported during the stable phase. So I disagree with this conclusion, we don't need a lot of people using it. Only a few that care about it.

> should we mark every new feature as unstable_ (we used to do this currently to have a space for breaking the API if needed

I think only when we see purpose in this https://mui.com/versions/#what-doesnt-count-as-a-breaking-change.

Reply:
> We plan to remove the prefix in v8 anyways.

Great, it's definitely more about setting up some baseline on how we handle this at scale, than fixing a specific problem. 

Alright, I'm merging, we will revisit if we discover that not enough people try to "Preview" features. Thanks for the feedback